internal/process/provisioning/create_runtime_resource_step.go

```diff
@@ -228,6 +228,24 @@ func (s *CreateRuntimeResourceStep) providerValues(operation *internal.Operation
 			MultiZone:              s.config.MultiZoneCluster,
 			ProvisioningParameters: operation.ProvisioningParameters,
 		}
+	case broker.FreemiumPlanID:
+		var freemiumProvider internal.CloudProvider
+		if operation.ProvisioningParameters.Parameters.Provider == nil {
+			//TODO - default provider?
+			return provider.Values{}, fmt.Errorf("provider not definded for freemium plan")
+		} else {
```

Comment:
Can we move it outside of else? Because of return it is not necessary.

Reply:
`else` removed in the next push.